src/react-components/src/components/01-atoms/Forms/Input.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from "react";
+
+import Input from "./Input";
+
+export default {
+  title: "Input",
+  component: Input,
+};
+
+export const input = () => <Input labelId="blah"
```

Comment:
These examples don't work because you're adding the same `ID` for both the label and input element, so now there's the a11y issue of id redundancy.